src/c_type_generator.py

```diff
@@ -230,10 +255,13 @@ def c_type_from_nodeset(self,
                 rv = ArrayType(rv, count)
             elif count == DerefLabel.COUNT_NULLTERM:
                 # C type for null terminated string is [w]char[_t]*
-                if rv.size in (1, 2, 4): # Valid character sizes
+                if rv.size in (1, 2, 4):  # Valid character sizes
                     rv = CharType(rv.size)
                 else:
-                    self.info("Unexpected character size for null-terminated string: %d", rv.size)
+                    self.info(
+                        "Unexpected character size for null-terminated string: %d",
```

Comment:
Is there a reason this code base is using format strings instead of f-strings? Not a big deal; mostly curiosity.

Reply:
This is why https://medium.com/swlh/why-it-matters-how-you-log-in-python-1a1085851205

Reply:
Makes sense.